queries/r/textobjects.scm

```diff
@@ -79,44 +67,44 @@
   .
   (_) @parameter.inner
   .
-  ","? @_end)
+  (comma)? @_end)
   (#make-range! "parameter.outer" @parameter.inner @_end))
 
 ; number
 (float) @number.inner
 
 ; assignment
-(left_assignment
+("<-"
```

Comment:
Okay this changes the syntax of these queries to be no longer parent-child but rather sequential nodes. Is this intended? @PMassicotte

What I mean by the above is
Before, the query would look for `name/value` field that are direct children of `left_assignment`, but now it looks for `name/value` field that have the same direct parent as `<-`

Reply:
> Okay this changes the syntax of these queries to be no longer parent-child but rather sequential nodes. Is this intended? @PMassicotte

I am not a treesitter expert by no mean, but this is my understanding.